src/components/indicators/z-stepper-item/index.tsx

```diff
@@ -24,33 +24,40 @@ export class ZStepperItem {
   @Prop({attribute: "pressed"})
   pressed: boolean;
 
+  /**
+   * The checked state of the item.
+   */
+  @Prop({attribute: "checked"})
+  checked: boolean;
+
   /**
    * The disabled state of the item.
    */
   @Prop({attribute: "disabled"})
   disabled: boolean;
 
-  render(): HTMLAnchorElement | HTMLButtonElement {
-    if (this.href) {
-      return (
-        <a
-          href={!this.disabled && this.href}
-          class="stepper-item"
-        >
-          <div class="indicator">{this.index}</div>
-          <span>
-            <slot />
-          </span>
-        </a>
-      );
-    }
+  private getAttributes(): Record<string, unknown> {
+    const href = this.href && !this.pressed && !this.disabled ? {onClick: () => (location.href = this.href)} : {};
```

Comment:
Come mai `: {}` e non `: undefined`?

Reply:
Nessun motivo i particolare. Ho allineato la gestione degli attribute come fatto nella `z-input`. 
Anche lì avevo usato `{}` in  caso di attribute non presente. 
Ma direi che si può tranquillamente mettere `undefined`